<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>635</width>
    <height>500</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>600</width>
    <height>500</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Visor de Im√°genes</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_5">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <layout class="QHBoxLayout" name="accionesLayaout">
          <item>
           <widget class="QPushButton" name="abrirButton">
            <property name="text">
             <string>Abrir</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="cerrarButton">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>Cerrar</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="guardarButton">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>Guardar</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <property name="sizeConstraint">
           <enum>QLayout::SetDefaultConstraint</enum>
          </property>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="verLayout">
          <item>
           <widget class="QPushButton" name="zoomOutButton">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>-</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="zoomZeroButton">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>100%</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="zoomInButton">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>+</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <widget class="GraphicsView" name="graphicsView"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>635</width>
     <height>27</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuArchivo">
    <property name="title">
     <string>Archivo</string>
    </property>
    <addaction name="actionAbrir"/>
    <addaction name="separator"/>
    <addaction name="actionGuardar"/>
    <addaction name="actionGuardar_como"/>
    <addaction name="separator"/>
    <addaction name="actionCerrar"/>
    <addaction name="actionSalir"/>
   </widget>
   <widget class="QMenu" name="menuEditar">
    <property name="title">
     <string>Editar</string>
    </property>
    <addaction name="actionAbrir_con"/>
    <addaction name="separator"/>
    <addaction name="actionAbrir_directorio"/>
    <addaction name="actionAbrir_terminal"/>
    <addaction name="separator"/>
    <addaction name="actionRenombrar"/>
    <addaction name="actionEliminar"/>
   </widget>
   <widget class="QMenu" name="menuHerramientas">
    <property name="title">
     <string>Herramientas</string>
    </property>
    <widget class="QMenu" name="menuZoom">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="title">
      <string>Zoom</string>
     </property>
     <addaction name="actionAmpliar"/>
     <addaction name="actionReducir"/>
     <addaction name="actionAjustar"/>
     <addaction name="actionNormal"/>
    </widget>
    <widget class="QMenu" name="menuRotar">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="title">
      <string>Rotar</string>
     </property>
     <addaction name="actionRotarDerecha"/>
     <addaction name="actionRotarIzquierda"/>
    </widget>
    <addaction name="menuZoom"/>
    <addaction name="menuRotar"/>
    <addaction name="separator"/>
    <addaction name="actionRestablecer"/>
   </widget>
   <widget class="QMenu" name="menuPreferencias">
    <property name="title">
     <string>Preferencias</string>
    </property>
    <addaction name="actionPantalla_completa"/>
   </widget>
   <widget class="QMenu" name="menuAyuda">
    <property name="title">
     <string>Ayuda</string>
    </property>
    <addaction name="actionAyuda"/>
    <addaction name="separator"/>
    <addaction name="actionAcerca_de"/>
   </widget>
   <addaction name="menuArchivo"/>
   <addaction name="menuEditar"/>
   <addaction name="menuHerramientas"/>
   <addaction name="menuPreferencias"/>
   <addaction name="menuAyuda"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionAyuda">
   <property name="text">
    <string>Ayuda</string>
   </property>
   <property name="shortcut">
    <string>F1</string>
   </property>
  </action>
  <action name="actionAcerca_de">
   <property name="text">
    <string>Acerca de</string>
   </property>
   <property name="shortcut">
    <string>F2</string>
   </property>
  </action>
  <action name="actionGuardar">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Guardar</string>
   </property>
  </action>
  <action name="actionGuardar_como">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Guardar como</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionCerrar">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Cerrar</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+W</string>
   </property>
  </action>
  <action name="actionSalir">
   <property name="text">
    <string>Salir</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionPantalla_completa">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Pantalla completa</string>
   </property>
   <property name="shortcut">
    <string>F11</string>
   </property>
  </action>
  <action name="actionAbrir_con">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Abrir con...</string>
   </property>
  </action>
  <action name="actionRenombrar">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Renombrar</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
  </action>
  <action name="actionEliminar">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Eliminar</string>
   </property>
   <property name="shortcut">
    <string>Del</string>
   </property>
  </action>
  <action name="actionAbrir_directorio">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Abrir directorio</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+D</string>
   </property>
   <property name="shortcutContext">
    <enum>Qt::WindowShortcut</enum>
   </property>
  </action>
  <action name="actionAbrir_terminal">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Abrir terminal</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+T</string>
   </property>
   <property name="shortcutContext">
    <enum>Qt::WindowShortcut</enum>
   </property>
  </action>
  <action name="actionRestablecer">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Restablecer</string>
   </property>
  </action>
  <action name="actionAmpliar">
   <property name="text">
    <string>Ampliar</string>
   </property>
   <property name="shortcut">
    <string>Ctrl++</string>
   </property>
  </action>
  <action name="actionReducir">
   <property name="text">
    <string>Reducir</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+-</string>
   </property>
  </action>
  <action name="actionAjustar">
   <property name="text">
    <string>Ajustar</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+=</string>
   </property>
  </action>
  <action name="actionNormal">
   <property name="text">
    <string>Normal</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+0</string>
   </property>
  </action>
  <action name="actionRotarDerecha">
   <property name="text">
    <string>Rotar a la derecha</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+L</string>
   </property>
  </action>
  <action name="actionRotarIzquierda">
   <property name="text">
    <string>Rotar a la izquierda</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+K</string>
   </property>
  </action>
  <action name="actionAbrir">
   <property name="text">
    <string>Abrir</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>GraphicsView</class>
   <extends>QGraphicsView</extends>
   <header location="global">graphicsview.hpp</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
